revup/revup.py

```diff
@@ -247,6 +197,18 @@ async def main() -> int:
         "config",
         add_help=False,
     )
+    toolkit_parser = subparsers.add_parser(
+        "toolkit", description="Test various subfunctionalities."
```

Comment:
I see this is just the same description `toolkit` had before, but is this really only for testing?  I've definitely used `toolkit` generally for real things